openapi/social/social.yaml

```diff
@@ -275,21 +361,57 @@ components:
             messageText:
               type: string
               description: 'The text of the message you’re making. This text is subject to limitations depending on what service you are posting to, such as Tweet length for Twitter. '
-            mediaUrls:
-              description: 'Links to pieces of media you want on the message you’re sending. These can be images, videos, and gifs. Media are subject to limitations depending on the social network you are posting to.'
+            status:
+              type: string
+              enum:
+                - scheduled
+                - posted
+              description: 'A string representing the tag assigned to a post.'
+              example: 'SCHEDULED'
+            media:
               type: array
+              description: 'An array containing the media attached to the post.'
               items:
-                type: string
-            sendAt:
+                type: object
+                properties:
+                  type:
+                    type: string
+                    enum:
+                      - image
+                      - video
+                      - gif
+                    example: 'image'
+                  url:
+                    type: string
+                    example: 'https://www.example.com/image.jpg'
+            postedAt:
               type: string
               format: date-time
               description: |-
-                The time you wish your message to be sent out.
-
-                Defaults to 5 minutes after the Message is made, and must be at least 5 minutes after the Message is created.
-
+                A string representing the timestamp at which the post was posted.
```

Comment:
We need to keep the text about 5 minute minimum when creating new messages.

Reply:
I don't see this restored yet.

Reply:
It looks like this got added to the postedAt field instead of the scheduledAt field.